drivers/postgres.go

```diff
@@ -0,0 +1,114 @@
+package drivers
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strings"
+)
+
+const (
+	PSQLDescribeTemplate   = "select column_name, data_type, character_maximum_length, column_default, is_nullable,numeric_precision,numeric_scale from INFORMATION_SCHEMA.COLUMNS where table_name = '%s'"
+	PSQLConnectionTemplate = "host=%s port=%s user=%s password=%s dbname=%s sslmode=disable"
+	PSQLInsertTemplate     = "INSERT INTO %s(\"%s\") VALUES(%s)"
+	PSQLShowTablesQuery    = "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname != 'pg_catalog' AND schemaname != 'information_schema';"
+)
+
+type Postgres struct {
+	f Flags
+}
+
+func (p Postgres) ShowTables(db *sql.DB) ([]string, error) {
+	rows, err := db.Query(PSQLShowTablesQuery)
+	if err != nil {
+		return nil, err
+	}
+	var tables []string
+	for rows.Next() {
+		var table string
+		rows.Scan(&table)
+		tables = append(tables, table)
+	}
+	return tables, nil
+}
+
+func (p Postgres) Connection() string {
+	return fmt.Sprintf(PSQLConnectionTemplate,
+		p.f.Host, p.f.Port, p.f.Username, p.f.Password, p.f.Database)
+}
+
+func (p Postgres) Driver() string {
+	return p.f.Driver
+}
+
+func (p Postgres) Insert(fields []string, table string) string {
+	return fmt.Sprintf(PSQLInsertTemplate, table, strings.Join(fields, "\",\""), pgValPlaceholder(len(fields)))
```

Comment:
Please use string's backtick semantic in strings.Join(...). I mean something like this:
```go
strings.Join(fields, `","`)
```
It looks much simple.